### ISSUE TITLE
Tests not detected with ASP.NET 5 Class Library

### ISSUE THREAD

**Issue by richardmatsu** [fixie/fixie#86]
Tried with Visual Studio test runner and the Console runner. The former simply does not detect any tests. If there's logs I don't know where they are. Within VS 2015 CTP 5, I am able to create a .NET 4.5 class library, create tests and Fixie detects and runs them as you'd expect. Likewise, the Console runner works with .NET 4.5 assemblies.

I also thought Fixie might work if I created .NET 4.5 outputs from the ASP.NET 5 Class Library project, but this also does not work.

With the Console runner, there is an assembly binding error:

Fatal Error: System.IO.FileNotFoundException: Could not load file or assembly 'Fixie, Version=1.0.0.3, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified. File name: 'Fixie, Version=1.0.0.3, Culture=neutral, PublicKeyToken=null'

Turning on the assembly bind failure logging gives additional, confusing information (at least to me):

=== Pre-bind state information ===
LOG: DisplayName = Fixie, Version=1.0.0.3, Culture=neutral, PublicKeyToken=null
 (Fully-specified)
LOG: Appbase = file:///C:/somePath/Visual Studio 2015/Projects/Sample.Web/artifacts/bin/Tests/Debug/net45
LOG: Initial PrivatePath = NULL
Calling assembly : (Unknown).
===
LOG: This bind starts in default load context.
LOG: Found application configuration file (c:\projectPath\packages\Fixie.1.0.0.3\lib\net45\Fixie.Console.exe.Config).
LOG: Using application configuration file: c:\projectPath\packages\Fixie.1.0.0.3\lib\net45\Fixie.Console.exe.Config
LOG: Using host configuration file:
LOG: Using machine configuration file from C:\Windows\Microsoft.NET\Framework\v4.0.30319\config\machine.config.
LOG: Policy not being applied to reference at this time (private, custom, partial, or location-based assembly bind).
LOG: Attempting download of new URL file:///C:/somePath/Visual Studio 2015/Projects/Sample.Web/artifacts/bin/Tests/Debug/net45/Fixie.DLL.
LOG: Attempting download of new URL file:///C:/somePath/Visual Studio 2015/Projects/Sample.Web/artifacts/bin/Tests/Debug/net45/Fixie/Fixie.DLL.
LOG: Attempting download of new URL file:///C:/somePath/Visual Studio 2015/Projects/Sample.Web/artifacts/bin/Tests/Debug/net45/Fixie.EXE.
LOG: Attempting download of new URL file:///C:/somePath/Visual Studio 2015/Projects/Sample.Web/artifacts/bin/Tests/Debug/net45/Fixie/Fixie.EXE.

**Comment by plioi** [comment]
@richardmatsu and I already discussed this over twitter, but if anyone else is reading this, support for ASP.NET 5 is planned but not implemented yet.  Since ASP.NET 5 is a dramatic overhaul and is currently a moving target, we can't release something useful or stable for it yet.  The xUnit team is facing a similar challenge, and at the time of this writing they cannot proceed until more is released by the ASP.NET team.  See https://twitter.com/xunit/status/567775550433796096

**Comment by davidpeden3** [comment]
Looks like [progress has been made](http://xunit.github.io/release-notes/2015-03-21.html). Interested to see the same here. ;)

**Comment by lbargaoanu** [comment]
@plioi I can look into this if you're interested.

**Comment by plioi** [comment]
I'm going to close this as a duplicate of #145. ASP.NET Core support is really .NET Core support, and a great deal has changed there since this issue was first reported.
